### PR TITLE
Added debug query log for MongoDB #5600

### LINKED ISSUE #5600: Add debug query log for MongoDB
Related to https://stackoverflow.com/questions/64432527/how-to-intercept-mondodb-query-from-presto-connector

https://github.com/prestosql/presto/blob/80d88f9cd66125cf9ba8db06cd6e73d61104ef3a/presto-mongodb/src/main/java/io/prestosql/plugin/mongodb/MongoSession.java#L274-L303

### PR DESCRIPTION
**Pull request by shandhanumohan**
Fixes #5600

### CODE REVIEW

**Review by Praveen2112 — APPROVED** [review]
Thanks for working on this.

**Review by ebyhr — COMMENTED** [review]
Thanks for your quick fix. Please amend the commit message (≠PR title) as follows. 
```
Add debug query log for MongoDB
```
This is commit message format guideline.
https://github.com/prestosql/presto/blob/master/DEVELOPMENT.md#format-git-commit-messages

Also, please make sure the email of this commit.

File: presto-mongodb/src/main/java/io/prestosql/plugin/mongodb/MongoSession.java [hunk 1]
```diff
@@ -260,6 +260,7 @@ private MongoColumnHandle buildColumnHandle(Document columnMeta)
     {
         String schemaName = toRemoteSchemaName(schema);
         String tableName = toRemoteTableName(schemaName, table);
+        log.debug("Database: %s Collection: %s", schemaName, tableName);
```
[the remark this answers is not in the document]
Comment (shandhanumohan) [reply presto-mongodb/src/main/java/io/prestosql/plugin/mongodb/MongoSession.java]: @ebyhr Made amendments, please check.

**Review by ebyhr — APPROVED** [review]
Looks good. Please squash commits into one with `Add debug query log for MongoDB` title.

When you follow upstream, please use rebase, not merge.

**Comment by shandhanumohan** [comment]
> Looks good. Please squash commits into one with `Add debug query log for MongoDB` title.
> 
> When you follow upstream, please use rebase, not merge.

**Comment by shandhanumohan** [comment]
Sorry, accidentally closed the PR. Reopening it. @ebyhr commits have been squashed into `Add debug query log for MongoDB` title.

**Comment by ebyhr** [comment]
Could confirm again? > https://github.com/prestosql/presto/pull/5612/commits
Also, please remove commit body.

**Comment by shandhanumohan** [comment]
Removed commit body. @ebyhr, please check.

**Comment by ebyhr** [comment]
Merged, thanks!

**Pull request merged**